src/ProjectService.Business/Commands/Project/GetProjectCommand.cs

```diff
@@ -40,60 +42,26 @@ public class GetProjectCommand : IGetProjectCommand
 
         private DepartmentInfo GetDepartment(Guid departmentId, List<string> errors)
         {
-            string errorMessage = $"Can not get department info for DepartmentId '{departmentId}'. Please try again later.";
-
-            DepartmentInfo department = null;
-
             try
             {
-                var departmentResponse = _departmentRequestClient.GetResponse<IOperationResult<IGetDepartmentResponse>>(
-                    IGetDepartmentRequest.CreateObj(null, departmentId)).Result;
-
-                if (departmentResponse.Message.IsSuccess)
+                IOperationResult<IGetDepartmentResponse> departmentResponse = 
+                    _departmentRequestClient.GetResponse<IOperationResult<IGetDepartmentResponse>>
+                    (
+                        IGetDepartmentRequest.CreateObj(null, departmentId)
+                    )
+                    .Result.Message;
+
+                if (departmentResponse.IsSuccess)
                 {
-                    department = _departmentInfoMapper.Map(departmentResponse.Message.Body);
+                    return _departmentInfoMapper.Map(departmentResponse.Body);
                 }
```

Comment:
Please, add loger if !departmentResponse.IsSuccess and not exception

Reply:
++